draft-ietf-netmod-schema-mount.org

```diff
@@ -290,13 +281,16 @@ the mounted schema is specified as implementation-time data that can
 be retrieved together with YANG library data for the parent schema,
 i.e., even before any instances of the mount point exist. However, the
 mounted schema has to be the same for all instances of the mount
-point.
+point. Note, that in this case a mount point may included a mounted YANG
+library module and the data contained in the mounted module MUST exactly
+match the data contained in the "schema" entries associated with the
+mount point.
 
-Each entry of the "schema" list contains
+Each entry of the "schema" list contains:
 
 - a list in the YANG library format specifying all YANG modules
-  (and revisions etc.) that are implemented or imported in the mounted
-  schema;
+  (augmentations, and revisions etc.) that are implemented or imported
```

Comment:
= "modules that only contain augmentations"
Please restate as you see fit.  The objective was to cover the confusion that was discussed on the netmod list, so that the next reader wasn't confused...